dev/com.ibm.ws.jaxrs.2.1.sse_fat/test-applications/DelaySseApp/src/jaxrs21sse/delay/DelaySseTestServlet.java

```diff
@@ -88,14 +88,25 @@ public void accept(Throwable t) {
 
                                 @Override
                                 public void run() {
-                                    System.out.println("completion runnable executed");
-                                    String sourceString = source.toString();
-                                    int delayStringStart = sourceString.indexOf("delay=");
-                                    String delayString = sourceString.substring(delayStringStart);
-                                    int delaystart = delayString.indexOf("=") + 1;
-                                    int delaystop = delayString.indexOf("|");
-                                    eventSourceTimes.add(delayString.substring(delaystart, delaystop));
-                                    executionLatch.countDown();
+                                    try {
+                                        System.out.println("completion runnable executed");
+                                        String sourceString = source.toString();
+                                        System.out.println("completion runnable sourceString = " + sourceString);
+                                        int delayStringStart = sourceString.indexOf("delay=");
+                                        //RestEasy uses reconnectDelay not delay
+                                        if (delayStringStart == -1) {
+                                            delayStringStart = sourceString.indexOf("reconnectDelay");
+                                        }
+                                        String delayString = sourceString.substring(delayStringStart);
+                                        int delaystart = delayString.indexOf("=") + 1;
+                                        int delaystop = delayString.indexOf("|");
+                                        eventSourceTimes.add(delayString.substring(delaystart, delaystop));                                    
+                                        executionLatch.countDown();
+                                        System.out.println("latch counted down 1 and now is: " + executionLatch.getCount());
+                                    } catch (Throwable t) {
```

Comment:
Is this try/catch intended to be removed?  If not, the catch block should probably re-throw the exception it caught to be consistent with the previous behavior.

Reply:
added throw.